app/src/utils/formatters.ts

```diff
@@ -1,3 +1,15 @@
+export function formatTimeLimit(time: Date): string {
```

Comment:
We should add treatment when showing 1 hour, otherwise it'll say, 1 hours 30 minutes, instead of 1 hour 30 minutes.

Reply:
good catch! The same can be applied to the minutes, if we want to be perfectionists.